src/data.js

```diff
@@ -0,0 +1,1023 @@
+const scorecardData ={ 
```

Comment:
FYI - we should be wary of adding real data here - we should maybe make it obvious that this is mock data with a comment so that people peering into this repo don't get confused and think this is Sunrise's official stance

Reply:
Noted! Will make it clear that this is mock/dummy data.